filip/models/ngsi_v2/context.py

```diff
@@ -182,8 +183,20 @@ def __init__(self, id: str, type: Union[str, Enum] = None, **data):
                 # `type` was found and pydantic will raise the correct error
                 super().__init__(id=id, **data)
         # This will result in usual behavior
+        data.update(self._validate_attributes(data))
         super().__init__(id=id, type=type, **data)
 
+    # Validation of attributes
+    @classmethod
+    def _validate_attributes(cls, data: dict):
+        """
+        Validate attribute name and value of the entity in keyvalues format
+        """
+        for attr_name, attr_value in data.items():
+            validate_fiware_attribute_value_regex(attr_value)
+            validate_fiware_datatype_string_protect(attr_name)
+        return data
```

Comment:
@mwr-ebc FYI, I add this code block for `ContextEntityKeyValues`. Please let me know if this raises any error